source/crud/read.txt

```diff
@@ -0,0 +1,166 @@
+.. _mongosh-read:
+
+===============
+Query Documents
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+Use the :method:`db.collection.find()` method in the |mdb-shell|
+to query documents in a collection.
+
+.. include:: /includes/fact-sample-data-examples.rst
+
+.. TODO: Make a callout here to the Connect to an Atlas Cluster section
+   in the connection docs
+
+Select All Documents in a Collection
+------------------------------------
+
+To select all documents in the collection, pass an empty document as the
+query filter parameter to the find method. The query filter parameter
+determines the select criteria.
+
+.. example::
+
+   To return all documents from the ``sample_mflix.movies`` collection:
+
+   .. code-block:: javascript
+
+      use sample_mflix
+
+      db.movies.find()
+
+   This operation is equivalent to the following SQL statement:
+
+   .. code-block:: sql
+      :copyable: false
+
+      SELECT * FROM movies
+
+Specify Equality Condition
+--------------------------
+
+To select documents which match an equality condition, specify
+the condition as a ``<field>:<value>`` pair in the
+:manual:`query filter document </document/#query-filter-documents>`.
+
+.. example::
+
+   To return all movies where the ``title`` equals ``Titanic`` from the
+   ``sample_mflix.movies`` collection:
+
+   .. code-block:: javascript
+
+      use sample_mflix
+
+      db.movies.find( { "title": "Titanic" } )
+
+   This operation corresponds to the following SQL statement:
+
+   .. code-block:: sql
+      :copyable: false
+
+      SELECT * FROM movies WHERE title = "Titanic"
```

Comment:
Are these pulled from the manual? Do we have corresponding SQL commands for the commands in the other tutorials?

Reply:
don't think it's crucial to have these now.